src/components/LoadingIcon/LoadingIcon.tsx

```diff
@@ -17,16 +17,16 @@ const useStyles = makeStyles({
     }
   },
   orange: {
-    animation: 'bumpit 2s linear 0.66s infinite'
+    animation: `$bumpit 2s linear 0.66s infinite`
```

Comment:
Do these need to be string templates for the `$` keyframe reference to work?

Reply:
from what i have read and understand, yes!

There's another syntax that also works, but it's not an exact match to what we are using across our other components so i think using the string templates is the safest approach!